polar2grid/compare.py

```diff
@@ -277,6 +291,7 @@ def compare_files(self, file1, file2, file_type=None, **kwargs):
             # guess based on file extension
             ext = os.path.splitext(file1)[-1]
             file_type = file_ext_to_compare_func.get(ext)
+            file_type = file_ext_to_compare_func.get(ext)
```

Comment:
This is a duplicate of the line above it, right?

Reply:
Yes, not sure how that got duplicated.